### DISCUSSION TITLE
🐞 Failed  /message/{{ $json.id }}/update on @newsletter | Caption Removed

### DISCUSSION THREAD

**Discussion by willianjack0** [aldinokemal/go-whatsapp-web-multidevice#220] [Announcements]
### What happened?

I'm encountering an unexpected behavior when updating a message. Upon successfully sending an image with a caption using the /send/image endpoint, subsequent updates to the message via the /message/{message_id}/update endpoint result in the complete removal of the original caption, even though the request body only includes a modification to the 'message' field.
<img width="542" alt="Screenshot 2024-12-10 at 09 41 48" src="https://github.com/user-attachments/assets/0dffaefc-ec9a-405c-a6c1-db9f8b3e525b">

Expected Behavior:
The updated message should update the original caption while only changing to the text specified in the 'message' field.

Actual Behavior:
The original caption is completely removed, leaving the image whitout caption.

Notes:

The /message/{message_id}/update endpoint currently only supports updating the 'message' field and does not acept the 'caption' attribute.

### Reproduction Steps

```bash
Steps to Reproduce:

1 - Send an image with a caption using a POST request to /send/image with the following parameters:
Bash

curl -X POST http://localhost:3000/send/image \
-H "Content-Type: multipart/form-data" \
-F "phone=id@newsletter" \
-F "caption=caption that works fine" \
-F "view_once=false" \
-F "image=@file" \
-F "compress=false"

2 - Retrieve the message ID from the response.
3 - Send a POST request to /message/{message_id}/update with the following body:
Bash

curl -X POST http://localhost:300/message/{message_id}/update \
-H "Content-tpye: application/json" \
-d '{"phone":"id@newsletter","message":"this caption clear the old one"}'
```

### Relevant log output

```bash
/message/{message_id}/update
code
SUCCESS
message
Update message success id@newsletter (server timestamp: 2024-12-10 12:41:35 +0000 UTC)
results
message_id
MESSAGEID
status
Update message success id@newsletter (server timestamp: 2024-12-10 12:41:35 +0000 UTC)
```

### Application Version

v4.21.1

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
you cannot update caption using that endpoint

**Reply by willianjack0** [reply]
Which Endpoint do I need to use? 
Searching the API documentation I didn't find any other route. 
Sorry for the inconvenience 🥲
